### DISCUSSION TITLE
local setting

### DISCUSSION THREAD

**Discussion by strongman22255** [espnet/espnet#5375] [Q&A] [answered]
how to set the model in local?
"from espnet2.bin.tts_inference import Text2Speech"
this code is requiring the online data, but I want to try to set it locally

**Comment by b-flo** [comment, marked as answer]
If you want to use `Text2Speech` with your local model, you need to pass the model and config through `model_file` and `train_config`.
The different cases are explained in the class doc: https://github.com/espnet/espnet/blob/master/espnet2/bin/tts_inference.py#L37-L62

**Reply by strongman22255** [reply]
thanks for your help. I will try it.

**Reply by strongman22255** [reply]
THANKS ALOT!!!!!!!!!😍😍😍

**Reply by strongman22255** [reply]
IT WORKS.
